pkg/artifactory/resource_artifactory_local_docker_repository.go

```diff
@@ -76,10 +73,42 @@ func resourceArtifactoryLocalDockerV2Repository() *schema.Resource {
 }
 
 func resourceArtifactoryLocalDockerV1Repository() *schema.Resource {
+
+	const pkt = "docker"
+
+	var dockerV1LocalSchema = mergeSchema(getBaseLocalRepoSchema(pkt), map[string]*schema.Schema{
+		"max_unique_tags": {
+			Type:     schema.TypeInt,
+			Optional: true,
+			Computed: true,
+		},
+		"tag_retention": {
+			Type:     schema.TypeInt,
+			Computed: true,
+		},
+		"block_pushing_schema1": {
+			Type:     schema.TypeBool,
+			Computed: true,
+		},
+		"api_version": {
+			Type:     schema.TypeString,
+			Computed: true,
+		},
+	})
+
 	// this is necessary because of the pointers
 	skeema := mergeSchema(map[string]*schema.Schema{}, dockerV1LocalSchema)
-	for key, value := range dockerV2LocalSchema {
-		skeema[key].Description = value.Description
+
+	var unPackLocalDockerV1Repository = func(data *schema.ResourceData) (interface{}, string, error) {
+		repo := DockerLocalRepositoryParams{
+			LocalRepositoryBaseParams: unpackBaseRepo("local", data, "docker"),
```

Comment:
use your constant here

Reply:
Improved. variable packageType used to replace literal `docker` from all occurrences